src/Loader.php

```diff
@@ -374,6 +414,48 @@ public function setEnvironmentVariable($name, $value = null)
         $_SERVER[$name] = $value;
     }
 
+	/**
+     * Set an environment variable from a Docker secret.
+     *
+     * This is done using:
+     * - putenv,
+     * - $_ENV,
+     * - $_SERVER.
+     *
+     * The environment variable value is stripped of single and double quotes.
+     *
+     * @param string      $name
+     * @param string|null $value
+     *
+     * @return void
+     */
+    public function setDockerEnvironmentVariable($name)
```

Comment:
keep it dry; this should be incorporated withing `setEnvironmentVariable` . i.e. therein a test `isDockerEtc` (sic)

Reply:
Can someone please implement this change, so we can move forward with this PR?